protocol/testing/genesis.sh

```diff
@@ -137,18 +137,16 @@ function edit_genesis() {
 	dasel put -t string -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[0].name' -v 'Large-Cap'
 	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[0].initial_margin_ppm' -v '50000' # 5%
 	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[0].maintenance_fraction_ppm' -v '600000' # 60% of IM
-	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[0].base_position_notional' -v '1000000000000' # 1_000_000 USDC
 	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[0].impact_notional' -v '10000000000' # 10_000 USDC (500 USDC / 5%)
 	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[0].open_interest_lower_cap' -v '0' # OIMF doesn't apply to Large-Cap
 	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[0].open_interest_upper_cap' -v '0' # OIMF doesn't apply to Large-Cap
 
-	# Liquidity Tier: Mid-Cap
+	# Liquidity Tier: Small-Cap
```

Comment:
We seem to follow conventions that the liquidity tier list goes from Largest cap -> long tail. Can we keep Mid-cap at `1` instead of `5` below?

Reply:
The params for `1` aren't changing but wanted to rename to `small-cap` and create a new `mid-cap` tier so that markets currently on tier 1 aren't affected. But your point makes sense, we'd just have to put the following messages in a single proposal:
1. Rename liquidity tier 1 to mid-cap and change params, add small-cap as tier 5
2. Update all perpetuals with liquidity tier 1 to liquidity tier 5

@yujin512 what do you think?

Reply:
Agreed the ordering according to IMF makes sense! But if we follow the logic thoroughly, the ordering would be

- FX
- Large Cap
- Mid cap (new)
- Isolated
- Small cap (previously Mid cap)
- Long-tail
- Safety.

which means updating all perpetuals to be in the correct tier. Also wondering if it would be worth reordering every time we want to add new tiers, which will likely be in the direction of increasing leverage and therefore shifting the rest down.

Reply:
Ok agree that we should avoid changing the existing perpetual objects for simplicity/reduce room for error.

For my understanding, if a mainnet market is currently `Mid-cap`, do we want them to be `Mid-cap` or `Small-cap` after the change? Is it not the former?

Reply:
Current Mid-cap markets will now be renamed to Small-cap, with the params as before. A new Mid-cap liquidity tier will be created with a different set of param values (e.g. 20x leverage instead of 10x of the old mid-cap tier). See here: https://www.notion.so/dydx/Research-liquidity-recommendations-d3b96d1fde374158841490b93c32de48?pvs=4#bf9047f7f06e487687d879a0f290e60c

Reply:
Currently mid-cap markets should have the same param values as now. Only renaming because the new liquidity tier (ex: SOL) makes sense to be between large cap and small cap!